mslib/msui/mscolab.py

```diff
@@ -966,6 +966,32 @@ def delete_account(self, _=None):
             if r.status_code == 200 and json.loads(r.text)["success"] is True:
                 self.logout()
 
+    @verify_token_required
+    def editfull_name(self):
+        fullname, ok = QtWidgets.QInputDialog.getText(
+            self.ui,
+            self.ui.tr("Edit Full Name"),
+            self.ui.tr(
+                f"You're about to change the full name - '{self.active_operation_name}' "
```

Comment:
What is the reason to add here self.active_operation_name ? How is the users full name linked to an operation?

Reply:
i am confused, I need to collect the user's full name and nickname from the UI, send the data to the server, and update the UI with the response from the server. Is this the correct approach?

Reply:
Start with only adding the full name.

Reply:
My question is why is the users fullname related to the operation name?

[Screenshot_20241207_041506_GitHub.jpg](https://github.com/user-attachments/assets/5d8ba6d7-50b0-417f-ba81-6df81a4aae92)

Reply:
sorry.. it's not directly related.

Reply:
> Start with only adding the full name. See discussion by Matthias.

where can i find it?

Reply:
https://github.com/Open-MSS/MSS/pull/2575#issuecomment-2500246905